### DISCUSSION TITLE
openDTU zurücksetzen auf Auslieferungszustand

### DISCUSSION THREAD

**Discussion by Fokker-1** [tbnobody/OpenDTU#2422] [Q&A]
Guten Tag,

nach Ändern meines PV-Systems benötige ich meinen HM-800-2t und die zugehörige openDTU (ist ein Fertiggerät) nicht mehr und möchte diese beiden Teile daher abgeben, die stammen aus 2023-06. Die FW der openDTU ist aktuell.

Nun sollen in der openDTU natürlich keine persönlichen Daten verbleiben, also z. B. WLAN-Login, Netzwerk-Daten, eigenes Admin-Passwort in der openDTU ...

**Wie kann ich die openDTU auf den Werkszustand zurücksetzen?**
**Alterntiv wäre ggf. noch nur das reine Löschen dieser persönlichen Daten eine Option?**

Ich habe hier viel gesucht, aber nicht wirklich Passendes gefunden, bin in diesen Dingen aber auch eher Laie.

Ich bitte daher um Unterstützung.

Danke im Voraus und Grüße
Werner

**Comment by mroenne2022** [comment]
Eigentlich doch im Menü unter Einstellungen/Konfigurationsverwaltung zu finden.
Zurücksetzen auf Werkseinstellungen !?
Alternativ mit Webflasher völlig neu flashen. 
https://solar.metacontrol.eu/opendtu_webinstall/

**Comment by Fokker-1** [comment]
Hallo, 

danke für die Infos.

Bei der Konfig.-Verwalt. finde ich nichts - oder muss ich da unter "Wiederherstellen" eine config. json hochladen?
Wenn ja, welche denn? ich hätte noch eine vom voherigen FW-Stand.

![Confug-Verw_oBatt](https://github.com/user-attachments/assets/0694ee4d-90af-4a86-bdbf-53174a4f7693)

Danke und Grüße
Werner

**Reply by mroenne2022** [reply]
Musst ganz runter scrollen,  könnte sein das du dich garnicht an der DTU angemeldet hast?
![Screenshot_20241122_180620_Chrome](https://github.com/user-attachments/assets/f5ddbd03-3480-466e-bab5-cd0e5c0f0cc4)

**Comment by Fokker-1** [comment]
... uups, Bild ist von einer on Battery, bei der "normalen" aber gleiche Funktion, sorry

**Comment by Fokker-1** [comment]
... autsch, das habe ich da tatsächlich nicht gesehen, sorry.
Das ist das (mein) Problem auf dem kleinen Handy-Display, da vergesse ich öfter mal, wirklich komplett nach unten zu scrollen.

Aber vielen Dank, dann versuche ich das mal.

Das Neuanmelden geht dann genau so nach dem gleichen Verfahren, wie bei einem Neugerät - denke ich mal, also über AP usw.?

Grüße und schönes Wochenende
Werenr

**Reply by mroenne2022** [reply]
Davon geh ich mal aus das ALLES verschwunden ist , lediglich noch die Firmware blank. Ich ab das noch nie machen müssen, hab öfters mal kpl neu geflasht. 
Dann berichte mal anschließend.

**Comment by Fokker-1** [comment]
Danke noch mal.

Meinst Du mit ALLES, dass auch das Pin-Mapping gelöscht sein könnte?
Das wäre natürlich ein echtes Problem, denn wie soll der Käufer das wieder hin bekommen?

Oder kann ich danach dann im AP-Mode den pin-mapping.json wieder reinladen, so dass damit der "echte" Originalzustand mit Pin-Mapping wieder besteht?

Komplett neu flashen kann ich nicht, davon habe ich keine Ahnung.

Danke und Gruß
Werner

**Reply by mroenne2022** [reply]
Mit alles meine ich die persönlichen Einstellungen wie WLAN, Passwörter,Wechselrichter usw 
Die Firmware bleibt ja , selbst wenn das pinmapping weg ist dann kannst du das natürlich wieder hochladen

**Comment by Fokker-1** [comment]
... ich danke Dir.

Du schreibst:

"... selbst wenn das pinmapping weg ist dann kannst du das natürlich wieder hochladen"

Frage dazu:
Das geht auch schon, wenn ich NICHT schon wieder im WLAN bin sondern noch im AP-Mode bei der Erstkonfiguration?
Sorry für meine vielen Fragen.

Danke und Gruß
Werner

**Reply by mroenne2022** [reply]
Im AP Mode bist du ja auch mit deinem Laptop raus aus deinem Heimnetz und bist per WLAN mit der DTU verbunden. Insofern geh ich davon aus das die PIN mapping hochgeladen werden kann, bin mirsogar sehr sicher. Ich hab das aber noch nicht machen müssen,  insofern kann ich es nicht 100% ig versichern. Ich wäre da nicht so skeptisch und würde es so machen , oder wenn du dich nicht traust dann ebn alle Daten löschen und am Ende die WLAN daten auch und dann ist es auch erledigt.

**Comment by Fokker-1** [comment]
Hallo,

sorry für die späte Rückmeldung.

Das habe ich jetzt über "Werkseinstellungen wiederherstellen" so gemacht - hat ales funktioniert!

Die DTU ist danach auch wieder über den AP erreichbar und die Oberfläche wird dargestellt bis zum Menü zur Eingabe der WLAN-Daten. Da gehe ich dann natürlich nicht mehr weiter.

Noch mal danke für den primma Support!

Viele Grüße
Werner

**Reply by mroenne2022** [reply]
Na wunderbar, dann hat es ja geklappt !

**Comment by 11freizeit** [comment]
Servus zusammen,

hatte die DTU gebraucht gekauft und mich angemeldet etc...
Hat alles funktioniert. Dann kam ich auf die Idee alles auf Werkseinstellung zurückzusetzen, nun geht garnichts mehr.
Kann die DTU einrichten, ins WLan bringen, Wechselrichter hinterlegen.
Wenn ich dann auf "Live Daten" gehe sieht es wie folgt aus
![Fehlermeldung](https://github.com/user-attachments/assets/8f2c3706-790d-4978-85b7-99424bed3de6)

In der Konfigurationsverwaltung habe ich die Konfigdateien runtergeladen und wieder eingespielt ohne Erfolg
![Konfigurationsverwaltung](https://github.com/user-attachments/assets/a7e249f2-0e0d-4c28-beec-46c1ea81d634)

Die DTU ist dunkel, zeigt keine Nullen etc. an.
Ich brauche keine Hinweise was ich hätte tun sollen oder nicht! Stand ist jetzt dafür benötige ich Hilfe!

Danke im vorraus und einen guten Start ins Wochenende

**Reply by GO113** [reply]
Hallo,
Wenn sie die pin_mapping Datei vor dem zurücksetzen auf Werkeinstellungen gesichert haben, müssen sie diese nach dem erneuten Hochladen in den Einstellungen unter Hardware auswählen und speichern. Wenn hier nur die Auswahl Standardeinstellung vorhanden ist, wird es schwierig die passende pin_mapping Datei zu erstellen, da jeder Hersteller verschiedene Pins auf dem ESP32 verwendet. Eine falsche pin_mapping kann zu eienem Totalabsturz führen und die DTU startet nicht mehr. 

Lg 
gerhard.obermueller@aon.at
